bigquery_etl/shredder/config.py

```diff
@@ -0,0 +1,239 @@
+
+DELETE_TARGETS = {
+    client_id_target(
+        table="search_derived.mobile_search_clients_daily_v1"
+    ): CLIENT_ID_SRC,
+    client_id_target(table="search_derived.search_clients_daily_v8"): CLIENT_ID_SRC,
+    client_id_target(table="search_derived.search_clients_last_seen_v1"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_derived.attitudes_daily_v1"): CLIENT_ID_SRC,
+    client_id_target(
+        table="telemetry_derived.clients_daily_histogram_aggregates_v1"
+    ): CLIENT_ID_SRC,
+    client_id_target(
+        table="telemetry_derived.clients_daily_scalar_aggregates_v1"
+    ): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_derived.clients_daily_v6"): CLIENT_ID_SRC,
+    client_id_target(
+        table="telemetry_derived.clients_histogram_aggregates_v1"
+    ): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_derived.clients_last_seen_v1"): CLIENT_ID_SRC,
+    client_id_target(
+        table="telemetry_derived.clients_profile_per_install_affected_v1"
+    ): CLIENT_ID_SRC,
+    client_id_target(
+        table="telemetry_derived.clients_scalar_aggregates_v1"
+    ): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_derived.core_clients_daily_v1"): CLIENT_ID_SRC,
+    client_id_target(
+        table="telemetry_derived.core_clients_last_seen_v1"
+    ): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_derived.core_live"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_derived.event_events_v1"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_derived.events_live"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_derived.experiments_v1"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_derived.main_events_v1"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_derived.main_summary_v4"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.block_autoplay_v1"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.crash_v4"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.downgrade_v4"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.event_v4"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.first_shutdown_v4"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.focus_event_v1"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.frecency_update_v4"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.health_v4"): CLIENT_ID_SRC,
+    client_id_target(table="telemetry_stable.heartbeat_v4"): CLIENT_ID_SRC,
+    client_id_target(
+        table="telemetry_stable.main_v4",
+        cluster_conditions=tuple(
+            (condition, needs_clustering)
+            for condition, needs_clustering in chain(
+                {
+                    f"sample_id = {sample_id} AND normalized_channel = 'release'": False
+                    for sample_id in range(100)
+                }.items(),
+                [("normalized_channel != 'release'", True)],
```

Comment:
We may want to be careful about whether `normalized_channel` is ever `NULL`, which IIRC, won't be caught by any of these cases. Even if `normalized_channel` _should_ never be null, it probably will be at some point.